src/introducao.md

```diff
@@ -1 +1,25 @@
 # Introdução
+
+Nix é uma ferramenta que aplica conceitos e questões da programação funcional em transformação de arquivos.
+Um dos títulos que dão ao Nix é que ele é um gerenciador de pacotes mas a base dele permite que ele seja mais.
+
+O Nix em sí é uma linguagem de programação de tipagem dinâmica que segue o paradigma funcional e seus conceitos mas que possui um diferencial que
+muda totalmente o jogo: O conceito de derivation.
+
+# O que são derivations?
+
+Derivations são o conceito de funções puras aplicado a arquivos. Enquanto que em outras linguagens, e no Nix também, os objetos
+manipulados por estas linguagens estão guardados na memória RAM para processamento o Nix tem como operar com transformações
+em arquivos.
```

Comment:
- O que acha de deixarmos os termos estrangeiros em itálico?
- Não entendi o fim do parágrafo, pode explicar?

```suggestion
_Derivations_ são o conceito de funções puras aplicado a arquivos. Assim como em outras linguagens, os objetos
manipulados estão guardados na memória RAM para processamento. 
```

Reply:
Dei um ajuste no fim do parágrafo